R/req-cookies.R

```diff
@@ -0,0 +1,38 @@
+#' Preserve cookies across requests
+#'
+#' By default, httr2 uses a clean slate for every request meaning that cookies
+#' are not automatically preserved across requests. To preserve cookies, you
+#' must set a cookie file which will be read before and updated after each
+#' request.
+#'
+#' @inheritParams req_perform
+#' @param path A path to file where cookies will be saved.
+#' @export
+#' @examples
+#' path <- tempfile()
+#' httpbin <- request(example_url()) %>%
+#'   req_cookie_file(path)
+#'
+#' # Manually set two cookies
+#' httpbin %>%
+#'   req_template("/cookies/set/:name/:value", name = "chocolate", value = "chip") %>%
+#'   req_perform() %>%
+#'   resp_body_json()
+#'
+#' httpbin %>%
+#'   req_template("/cookies/set/:name/:value", name = "oatmeal", value = "raisin") %>%
+#'   req_perform() %>%
+#'   resp_body_json()
+#'
+#' # The cookie path has a straightforward format
+#' readLines(path)
```

Comment:
maybe use `readChar()` + `cat()` for nicer printing?

```suggestion
cat(readChar(path, nchars = 1e4))
```